proto/slinky/oracle/v1/query.proto

```diff
@@ -36,14 +36,8 @@ message GetAllCurrencyPairsResponse {
 // GetPriceRequest either takes a CurrencyPair, or an identifier for the
 // CurrencyPair in the format base/quote
 message GetPriceRequest {
-  // CurrencyPairSelector represents the types that the user may provide to the
-  // request to identify a CurrencyPair
-  oneof currency_pair_selector {
-    // CurrencyPair type
-    CurrencyPair currency_pair = 1;
-    // string representation of the CurrencyPair
-    string currency_pair_id = 2;
-  }
+  // CurrencyPairID is the string representation of the CurrencyPair.
+  string currency_pair_id = 2;
```

Comment:
Query refactor removes the use of `oneof` and moves to just always query by CurrencyPairID ("ticker")

Reply:
maybe `Ticker` makes more sense than `CurrencyPairID`?

Reply:
gets confusing when considering that there is a separate ID for each CP